server/@types/express/index.d.ts

```diff
@@ -5,15 +5,19 @@ export declare module 'express-session' {
     searchType: string
     firstName?: string
     lastName?: string
-    dob: string
-    ageRange?: string
+    'dob-day'?: string
```

Comment:
seems odd field names - would it be better as `dobDay` etc?

Reply:
I didn't realise it was configurable - from govukDateInput - I'll have a look